{"user":"x_finder3","timestamp":"2024-09-03T10:31:44.782938","nav_name":"api_owasp_sen_data","action":"启动出错","params":"PubShm","operate_result":"错误","failed_reason":"[Errno 2] No such file or directory: '\/xlink_shm_httpub_2.pub'"}
{"user":"x_finder3","timestamp":"2024-09-03T10:31:44.784306","nav_name":"api_owasp_sen_data","action":"主函数","params":"异常退出PubShm:[Errno 2] No such file or directory: '\/xlink_shm_httpub_2.pub'","operate_result":"错误","failed_reason":"错误跟踪Traceback (most recent call last):\n  File \"xlink_core.pyx\", line 820, in xlink_core.Main5\n  File \"\/opt\/fbi-base\/lib\/python3.11\/multiprocessing\/shared_memory.py\", line 104, in __init__\n    self._fd = _posixshmem.shm_open(\n               ^^^^^^^^^^^^^^^^^^^^^\nFileNotFoundError: [Errno 2] No such file or directory: '\/xlink_shm_httpub_2.pub'\n\nDuring handling of the above exception, another exception occurred:\n\nTraceback (most recent call last):\n  File \"xlink_core.pyx\", line 1453, in xlink_core.x_start\n  File \"xlink_core.pyx\", line 830, in xlink_core.Main5\nException: PubShm:[Errno 2] No such file or directory: '\/xlink_shm_httpub_2.pub'\n"}
{"user":"x_finder3","timestamp":"2024-09-03T10:31:44.784436","nav_name":"api_owasp_sen_data","action":"停止","params":"二阶段开始停止!","operate_result":"信息","failed_reason":""}
{"user":"x_finder3","timestamp":"2024-09-03T10:31:44.784540","nav_name":"api_owasp_sen_data","action":"停止","params":"三阶段开始停止!","operate_result":"信息","failed_reason":""}
{"user":"x_finder3","timestamp":"2024-09-03T10:31:44.881875","nav_name":"api_owasp_sen_data","action":"停止","params":"数据窗口任务停止完毕!","operate_result":"信息","failed_reason":""}
{"user":"x_finder3","timestamp":"2024-09-03T10:31:45.282020","nav_name":"api_owasp_sen_data","action":"停止","params":"系统定时任务st_10s停止完毕!","operate_result":"信息","failed_reason":""}
{"user":"x_finder3","timestamp":"2024-09-03T10:31:49.782155","nav_name":"api_owasp_sen_data","action":"监控程序","params":"printf::收到消息退出","operate_result":"信息","failed_reason":""}
{"user":"x_finder3","timestamp":"2024-09-03T10:31:49.782742","nav_name":"api_owasp_sen_data","action":"停止","params":"全部停止完毕!","operate_result":"信息","failed_reason":""}
{"user":"x_finder3","timestamp":"2024-09-03T10:32:44.787152","nav_name":"api_owasp_sen_data","action":"启动参数","params":"shm_name: httpub,count: 8,","operate_result":"信息","failed_reason":""}
{"user":"x_finder3","timestamp":"2024-09-03T10:32:44.795572","nav_name":"api_owasp_sen_data","action":"组件启动","params":"开始验证授权....","operate_result":"信息","failed_reason":""}
{"user":"x_finder3","timestamp":"2024-09-03T10:32:44.795938","nav_name":"api_owasp_sen_data","action":"组件启动","params":"授权验证完成!","operate_result":"信息","failed_reason":""}
{"user":"x_finder3","timestamp":"2024-09-03T10:32:44.797103","nav_name":"api_owasp_sen_data","action":"组件启动","params":"数据窗口函数-3, 定时任务函数-0","operate_result":"信息","failed_reason":""}
{"user":"x_finder3","timestamp":"2024-09-03T10:32:44.797273","nav_name":"api_owasp_sen_data","action":"事件主循环","params":"PubShm 建立中","operate_result":"信息","failed_reason":""}
{"user":"x_finder3","timestamp":"2024-09-03T10:32:44.798124","nav_name":"api_owasp_sen_data","action":"启动出错","params":"PubShm","operate_result":"错误","failed_reason":"[Errno 2] No such file or directory: '\/xlink_shm_httpub_2.pub'"}
{"user":"x_finder3","timestamp":"2024-09-03T10:32:44.799643","nav_name":"api_owasp_sen_data","action":"主函数","params":"异常退出PubShm:[Errno 2] No such file or directory: '\/xlink_shm_httpub_2.pub'","operate_result":"错误","failed_reason":"错误跟踪Traceback (most recent call last):\n  File \"xlink_core.pyx\", line 820, in xlink_core.Main5\n  File \"\/opt\/fbi-base\/lib\/python3.11\/multiprocessing\/shared_memory.py\", line 104, in __init__\n    self._fd = _posixshmem.shm_open(\n               ^^^^^^^^^^^^^^^^^^^^^\nFileNotFoundError: [Errno 2] No such file or directory: '\/xlink_shm_httpub_2.pub'\n\nDuring handling of the above exception, another exception occurred:\n\nTraceback (most recent call last):\n  File \"xlink_core.pyx\", line 1453, in xlink_core.x_start\n  File \"xlink_core.pyx\", line 830, in xlink_core.Main5\nException: PubShm:[Errno 2] No such file or directory: '\/xlink_shm_httpub_2.pub'\n"}
{"user":"x_finder3","timestamp":"2024-09-03T10:32:44.799778","nav_name":"api_owasp_sen_data","action":"停止","params":"二阶段开始停止!","operate_result":"信息","failed_reason":""}
{"user":"x_finder3","timestamp":"2024-09-03T10:32:44.799878","nav_name":"api_owasp_sen_data","action":"停止","params":"三阶段开始停止!","operate_result":"信息","failed_reason":""}
{"user":"x_finder3","timestamp":"2024-09-03T10:32:44.896862","nav_name":"api_owasp_sen_data","action":"停止","params":"数据窗口任务停止完毕!","operate_result":"信息","failed_reason":""}
{"user":"x_finder3","timestamp":"2024-09-03T10:32:45.297033","nav_name":"api_owasp_sen_data","action":"停止","params":"系统定时任务st_10s停止完毕!","operate_result":"信息","failed_reason":""}
{"user":"x_finder3","timestamp":"2024-09-03T10:32:49.797215","nav_name":"api_owasp_sen_data","action":"监控程序","params":"printf::收到消息退出","operate_result":"信息","failed_reason":""}
{"user":"x_finder3","timestamp":"2024-09-03T10:32:49.797567","nav_name":"api_owasp_sen_data","action":"停止","params":"全部停止完毕!","operate_result":"信息","failed_reason":""}
{"user":"x_finder3","timestamp":"2024-09-03T10:33:44.890266","nav_name":"api_owasp_sen_data","action":"启动参数","params":"shm_name: httpub,count: 8,","operate_result":"信息","failed_reason":""}
{"user":"x_finder3","timestamp":"2024-09-03T10:33:44.897522","nav_name":"api_owasp_sen_data","action":"组件启动","params":"开始验证授权....","operate_result":"信息","failed_reason":""}
{"user":"x_finder3","timestamp":"2024-09-03T10:33:44.897949","nav_name":"api_owasp_sen_data","action":"组件启动","params":"授权验证完成!","operate_result":"信息","failed_reason":""}
{"user":"x_finder3","timestamp":"2024-09-03T10:33:44.899123","nav_name":"api_owasp_sen_data","action":"组件启动","params":"数据窗口函数-3, 定时任务函数-0","operate_result":"信息","failed_reason":""}
{"user":"x_finder3","timestamp":"2024-09-03T10:33:44.899319","nav_name":"api_owasp_sen_data","action":"事件主循环","params":"PubShm 建立中","operate_result":"信息","failed_reason":""}
{"user":"x_finder3","timestamp":"2024-09-03T10:33:44.900061","nav_name":"api_owasp_sen_data","action":"启动出错","params":"PubShm","operate_result":"错误","failed_reason":"[Errno 2] No such file or directory: '\/xlink_shm_httpub_2.pub'"}
{"user":"x_finder3","timestamp":"2024-09-03T10:33:44.901290","nav_name":"api_owasp_sen_data","action":"主函数","params":"异常退出PubShm:[Errno 2] No such file or directory: '\/xlink_shm_httpub_2.pub'","operate_result":"错误","failed_reason":"错误跟踪Traceback (most recent call last):\n  File \"xlink_core.pyx\", line 820, in xlink_core.Main5\n  File \"\/opt\/fbi-base\/lib\/python3.11\/multiprocessing\/shared_memory.py\", line 104, in __init__\n    self._fd = _posixshmem.shm_open(\n               ^^^^^^^^^^^^^^^^^^^^^\nFileNotFoundError: [Errno 2] No such file or directory: '\/xlink_shm_httpub_2.pub'\n\nDuring handling of the above exception, another exception occurred:\n\nTraceback (most recent call last):\n  File \"xlink_core.pyx\", line 1453, in xlink_core.x_start\n  File \"xlink_core.pyx\", line 830, in xlink_core.Main5\nException: PubShm:[Errno 2] No such file or directory: '\/xlink_shm_httpub_2.pub'\n"}
{"user":"x_finder3","timestamp":"2024-09-03T10:33:44.901427","nav_name":"api_owasp_sen_data","action":"停止","params":"二阶段开始停止!","operate_result":"信息","failed_reason":""}
{"user":"x_finder3","timestamp":"2024-09-03T10:33:44.901534","nav_name":"api_owasp_sen_data","action":"停止","params":"三阶段开始停止!","operate_result":"信息","failed_reason":""}
{"user":"x_finder3","timestamp":"2024-09-03T10:33:44.998978","nav_name":"api_owasp_sen_data","action":"停止","params":"数据窗口任务停止完毕!","operate_result":"信息","failed_reason":""}
{"user":"x_finder3","timestamp":"2024-09-03T10:33:45.399101","nav_name":"api_owasp_sen_data","action":"停止","params":"系统定时任务st_10s停止完毕!","operate_result":"信息","failed_reason":""}
{"user":"x_finder3","timestamp":"2024-09-03T10:33:49.899239","nav_name":"api_owasp_sen_data","action":"监控程序","params":"printf::收到消息退出","operate_result":"信息","failed_reason":""}
{"user":"x_finder3","timestamp":"2024-09-03T10:33:49.899750","nav_name":"api_owasp_sen_data","action":"停止","params":"全部停止完毕!","operate_result":"信息","failed_reason":""}
{"user":"x_finder3","timestamp":"2024-09-03T10:34:45.331595","nav_name":"api_owasp_sen_data","action":"启动参数","params":"shm_name: httpub,count: 8,","operate_result":"信息","failed_reason":""}
{"user":"x_finder3","timestamp":"2024-09-03T10:34:45.337399","nav_name":"api_owasp_sen_data","action":"组件启动","params":"开始验证授权....","operate_result":"信息","failed_reason":""}
{"user":"x_finder3","timestamp":"2024-09-03T10:34:45.337751","nav_name":"api_owasp_sen_data","action":"组件启动","params":"授权验证完成!","operate_result":"信息","failed_reason":""}
{"user":"x_finder3","timestamp":"2024-09-03T10:34:45.338683","nav_name":"api_owasp_sen_data","action":"组件启动","params":"数据窗口函数-3, 定时任务函数-0","operate_result":"信息","failed_reason":""}
{"user":"x_finder3","timestamp":"2024-09-03T10:34:45.338830","nav_name":"api_owasp_sen_data","action":"事件主循环","params":"PubShm 建立中","operate_result":"信息","failed_reason":""}
{"user":"x_finder3","timestamp":"2024-09-03T10:34:45.339407","nav_name":"api_owasp_sen_data","action":"启动出错","params":"PubShm","operate_result":"错误","failed_reason":"[Errno 2] No such file or directory: '\/xlink_shm_httpub_2.pub'"}
{"user":"x_finder3","timestamp":"2024-09-03T10:34:45.340168","nav_name":"api_owasp_sen_data","action":"主函数","params":"异常退出PubShm:[Errno 2] No such file or directory: '\/xlink_shm_httpub_2.pub'","operate_result":"错误","failed_reason":"错误跟踪Traceback (most recent call last):\n  File \"xlink_core.pyx\", line 820, in xlink_core.Main5\n  File \"\/opt\/fbi-base\/lib\/python3.11\/multiprocessing\/shared_memory.py\", line 104, in __init__\n    self._fd = _posixshmem.shm_open(\n               ^^^^^^^^^^^^^^^^^^^^^\nFileNotFoundError: [Errno 2] No such file or directory: '\/xlink_shm_httpub_2.pub'\n\nDuring handling of the above exception, another exception occurred:\n\nTraceback (most recent call last):\n  File \"xlink_core.pyx\", line 1453, in xlink_core.x_start\n  File \"xlink_core.pyx\", line 830, in xlink_core.Main5\nException: PubShm:[Errno 2] No such file or directory: '\/xlink_shm_httpub_2.pub'\n"}
{"user":"x_finder3","timestamp":"2024-09-03T10:34:45.340287","nav_name":"api_owasp_sen_data","action":"停止","params":"二阶段开始停止!","operate_result":"信息","failed_reason":""}
{"user":"x_finder3","timestamp":"2024-09-03T10:34:45.340386","nav_name":"api_owasp_sen_data","action":"停止","params":"三阶段开始停止!","operate_result":"信息","failed_reason":""}
{"user":"x_finder3","timestamp":"2024-09-03T10:34:45.438542","nav_name":"api_owasp_sen_data","action":"停止","params":"数据窗口任务停止完毕!","operate_result":"信息","failed_reason":""}
{"user":"x_finder3","timestamp":"2024-09-03T10:34:45.838659","nav_name":"api_owasp_sen_data","action":"停止","params":"系统定时任务st_10s停止完毕!","operate_result":"信息","failed_reason":""}
{"user":"x_finder3","timestamp":"2024-09-03T10:34:50.338802","nav_name":"api_owasp_sen_data","action":"监控程序","params":"printf::收到消息退出","operate_result":"信息","failed_reason":""}
{"user":"x_finder3","timestamp":"2024-09-03T10:34:50.339251","nav_name":"api_owasp_sen_data","action":"停止","params":"全部停止完毕!","operate_result":"信息","failed_reason":""}
{"user":"x_finder3","timestamp":"2024-09-03T10:35:45.612052","nav_name":"api_owasp_sen_data","action":"启动参数","params":"shm_name: httpub,count: 8,","operate_result":"信息","failed_reason":""}
{"user":"x_finder3","timestamp":"2024-09-03T10:35:45.619765","nav_name":"api_owasp_sen_data","action":"组件启动","params":"开始验证授权....","operate_result":"信息","failed_reason":""}
{"user":"x_finder3","timestamp":"2024-09-03T10:35:45.620190","nav_name":"api_owasp_sen_data","action":"组件启动","params":"授权验证完成!","operate_result":"信息","failed_reason":""}
{"user":"x_finder3","timestamp":"2024-09-03T10:35:45.621227","nav_name":"api_owasp_sen_data","action":"组件启动","params":"数据窗口函数-3, 定时任务函数-0","operate_result":"信息","failed_reason":""}
{"user":"x_finder3","timestamp":"2024-09-03T10:35:45.621402","nav_name":"api_owasp_sen_data","action":"事件主循环","params":"PubShm 建立中","operate_result":"信息","failed_reason":""}
{"user":"x_finder3","timestamp":"2024-09-03T10:35:45.622097","nav_name":"api_owasp_sen_data","action":"启动出错","params":"PubShm","operate_result":"错误","failed_reason":"[Errno 2] No such file or directory: '\/xlink_shm_httpub_2.pub'"}
{"user":"x_finder3","timestamp":"2024-09-03T10:35:45.623305","nav_name":"api_owasp_sen_data","action":"主函数","params":"异常退出PubShm:[Errno 2] No such file or directory: '\/xlink_shm_httpub_2.pub'","operate_result":"错误","failed_reason":"错误跟踪Traceback (most recent call last):\n  File \"xlink_core.pyx\", line 820, in xlink_core.Main5\n  File \"\/opt\/fbi-base\/lib\/python3.11\/multiprocessing\/shared_memory.py\", line 104, in __init__\n    self._fd = _posixshmem.shm_open(\n               ^^^^^^^^^^^^^^^^^^^^^\nFileNotFoundError: [Errno 2] No such file or directory: '\/xlink_shm_httpub_2.pub'\n\nDuring handling of the above exception, another exception occurred:\n\nTraceback (most recent call last):\n  File \"xlink_core.pyx\", line 1453, in xlink_core.x_start\n  File \"xlink_core.pyx\", line 830, in xlink_core.Main5\nException: PubShm:[Errno 2] No such file or directory: '\/xlink_shm_httpub_2.pub'\n"}
{"user":"x_finder3","timestamp":"2024-09-03T10:35:45.623443","nav_name":"api_owasp_sen_data","action":"停止","params":"二阶段开始停止!","operate_result":"信息","failed_reason":""}
{"user":"x_finder3","timestamp":"2024-09-03T10:35:45.623551","nav_name":"api_owasp_sen_data","action":"停止","params":"三阶段开始停止!","operate_result":"信息","failed_reason":""}
{"user":"x_finder3","timestamp":"2024-09-03T10:35:45.721082","nav_name":"api_owasp_sen_data","action":"停止","params":"数据窗口任务停止完毕!","operate_result":"信息","failed_reason":""}
{"user":"x_finder3","timestamp":"2024-09-03T10:35:46.121241","nav_name":"api_owasp_sen_data","action":"停止","params":"系统定时任务st_10s停止完毕!","operate_result":"信息","failed_reason":""}
{"user":"x_finder3","timestamp":"2024-09-03T10:35:50.621338","nav_name":"api_owasp_sen_data","action":"监控程序","params":"printf::收到消息退出","operate_result":"信息","failed_reason":""}
{"user":"x_finder3","timestamp":"2024-09-03T10:35:50.621902","nav_name":"api_owasp_sen_data","action":"停止","params":"全部停止完毕!","operate_result":"信息","failed_reason":""}
{"user":"x_finder3","timestamp":"2024-09-03T10:36:45.557548","nav_name":"api_owasp_sen_data","action":"启动参数","params":"shm_name: httpub,count: 8,","operate_result":"信息","failed_reason":""}
{"user":"x_finder3","timestamp":"2024-09-03T10:36:45.566123","nav_name":"api_owasp_sen_data","action":"组件启动","params":"开始验证授权....","operate_result":"信息","failed_reason":""}
{"user":"x_finder3","timestamp":"2024-09-03T10:36:45.566569","nav_name":"api_owasp_sen_data","action":"组件启动","params":"授权验证完成!","operate_result":"信息","failed_reason":""}
{"user":"x_finder3","timestamp":"2024-09-03T10:36:45.567727","nav_name":"api_owasp_sen_data","action":"组件启动","params":"数据窗口函数-3, 定时任务函数-0","operate_result":"信息","failed_reason":""}
{"user":"x_finder3","timestamp":"2024-09-03T10:36:45.567912","nav_name":"api_owasp_sen_data","action":"事件主循环","params":"PubShm 建立中","operate_result":"信息","failed_reason":""}
{"user":"x_finder3","timestamp":"2024-09-03T10:36:45.568616","nav_name":"api_owasp_sen_data","action":"启动出错","params":"PubShm","operate_result":"错误","failed_reason":"[Errno 2] No such file or directory: '\/xlink_shm_httpub_2.pub'"}
{"user":"x_finder3","timestamp":"2024-09-03T10:36:45.569891","nav_name":"api_owasp_sen_data","action":"主函数","params":"异常退出PubShm:[Errno 2] No such file or directory: '\/xlink_shm_httpub_2.pub'","operate_result":"错误","failed_reason":"错误跟踪Traceback (most recent call last):\n  File \"xlink_core.pyx\", line 820, in xlink_core.Main5\n  File \"\/opt\/fbi-base\/lib\/python3.11\/multiprocessing\/shared_memory.py\", line 104, in __init__\n    self._fd = _posixshmem.shm_open(\n               ^^^^^^^^^^^^^^^^^^^^^\nFileNotFoundError: [Errno 2] No such file or directory: '\/xlink_shm_httpub_2.pub'\n\nDuring handling of the above exception, another exception occurred:\n\nTraceback (most recent call last):\n  File \"xlink_core.pyx\", line 1453, in xlink_core.x_start\n  File \"xlink_core.pyx\", line 830, in xlink_core.Main5\nException: PubShm:[Errno 2] No such file or directory: '\/xlink_shm_httpub_2.pub'\n"}
{"user":"x_finder3","timestamp":"2024-09-03T10:36:45.570028","nav_name":"api_owasp_sen_data","action":"停止","params":"二阶段开始停止!","operate_result":"信息","failed_reason":""}
{"user":"x_finder3","timestamp":"2024-09-03T10:36:45.570135","nav_name":"api_owasp_sen_data","action":"停止","params":"三阶段开始停止!","operate_result":"信息","failed_reason":""}
{"user":"x_finder3","timestamp":"2024-09-03T10:36:45.667537","nav_name":"api_owasp_sen_data","action":"停止","params":"数据窗口任务停止完毕!","operate_result":"信息","failed_reason":""}
{"user":"x_finder3","timestamp":"2024-09-03T10:36:46.067717","nav_name":"api_owasp_sen_data","action":"停止","params":"系统定时任务st_10s停止完毕!","operate_result":"信息","failed_reason":""}
{"user":"x_finder3","timestamp":"2024-09-03T10:36:50.567851","nav_name":"api_owasp_sen_data","action":"监控程序","params":"printf::收到消息退出","operate_result":"信息","failed_reason":""}
{"user":"x_finder3","timestamp":"2024-09-03T10:36:50.568446","nav_name":"api_owasp_sen_data","action":"停止","params":"全部停止完毕!","operate_result":"信息","failed_reason":""}
{"user":"x_finder3","timestamp":"2024-09-03T10:37:45.885902","nav_name":"api_owasp_sen_data","action":"启动参数","params":"shm_name: httpub,count: 8,","operate_result":"信息","failed_reason":""}
{"user":"x_finder3","timestamp":"2024-09-03T10:37:45.893027","nav_name":"api_owasp_sen_data","action":"组件启动","params":"开始验证授权....","operate_result":"信息","failed_reason":""}
{"user":"x_finder3","timestamp":"2024-09-03T10:37:45.893394","nav_name":"api_owasp_sen_data","action":"组件启动","params":"授权验证完成!","operate_result":"信息","failed_reason":""}
{"user":"x_finder3","timestamp":"2024-09-03T10:37:45.894471","nav_name":"api_owasp_sen_data","action":"组件启动","params":"数据窗口函数-3, 定时任务函数-0","operate_result":"信息","failed_reason":""}
{"user":"x_finder3","timestamp":"2024-09-03T10:37:45.894620","nav_name":"api_owasp_sen_data","action":"事件主循环","params":"PubShm 建立中","operate_result":"信息","failed_reason":""}
{"user":"x_finder3","timestamp":"2024-09-03T10:37:45.895230","nav_name":"api_owasp_sen_data","action":"启动出错","params":"PubShm","operate_result":"错误","failed_reason":"[Errno 2] No such file or directory: '\/xlink_shm_httpub_2.pub'"}
{"user":"x_finder3","timestamp":"2024-09-03T10:37:45.896324","nav_name":"api_owasp_sen_data","action":"主函数","params":"异常退出PubShm:[Errno 2] No such file or directory: '\/xlink_shm_httpub_2.pub'","operate_result":"错误","failed_reason":"错误跟踪Traceback (most recent call last):\n  File \"xlink_core.pyx\", line 820, in xlink_core.Main5\n  File \"\/opt\/fbi-base\/lib\/python3.11\/multiprocessing\/shared_memory.py\", line 104, in __init__\n    self._fd = _posixshmem.shm_open(\n               ^^^^^^^^^^^^^^^^^^^^^\nFileNotFoundError: [Errno 2] No such file or directory: '\/xlink_shm_httpub_2.pub'\n\nDuring handling of the above exception, another exception occurred:\n\nTraceback (most recent call last):\n  File \"xlink_core.pyx\", line 1453, in xlink_core.x_start\n  File \"xlink_core.pyx\", line 830, in xlink_core.Main5\nException: PubShm:[Errno 2] No such file or directory: '\/xlink_shm_httpub_2.pub'\n"}
{"user":"x_finder3","timestamp":"2024-09-03T10:37:45.896460","nav_name":"api_owasp_sen_data","action":"停止","params":"二阶段开始停止!","operate_result":"信息","failed_reason":""}
{"user":"x_finder3","timestamp":"2024-09-03T10:37:45.896562","nav_name":"api_owasp_sen_data","action":"停止","params":"三阶段开始停止!","operate_result":"信息","failed_reason":""}
{"user":"x_finder3","timestamp":"2024-09-03T10:37:45.994287","nav_name":"api_owasp_sen_data","action":"停止","params":"数据窗口任务停止完毕!","operate_result":"信息","failed_reason":""}
{"user":"x_finder3","timestamp":"2024-09-03T10:37:46.394435","nav_name":"api_owasp_sen_data","action":"停止","params":"系统定时任务st_10s停止完毕!","operate_result":"信息","failed_reason":""}
{"user":"x_finder3","timestamp":"2024-09-03T10:37:50.894593","nav_name":"api_owasp_sen_data","action":"监控程序","params":"printf::收到消息退出","operate_result":"信息","failed_reason":""}
{"user":"x_finder3","timestamp":"2024-09-03T10:37:50.895083","nav_name":"api_owasp_sen_data","action":"停止","params":"全部停止完毕!","operate_result":"信息","failed_reason":""}
{"user":"x_finder3","timestamp":"2024-09-03T10:38:46.243782","nav_name":"api_owasp_sen_data","action":"启动参数","params":"shm_name: httpub,count: 8,","operate_result":"信息","failed_reason":""}
{"user":"x_finder3","timestamp":"2024-09-03T10:38:46.251104","nav_name":"api_owasp_sen_data","action":"组件启动","params":"开始验证授权....","operate_result":"信息","failed_reason":""}
{"user":"x_finder3","timestamp":"2024-09-03T10:38:46.251513","nav_name":"api_owasp_sen_data","action":"组件启动","params":"授权验证完成!","operate_result":"信息","failed_reason":""}
{"user":"x_finder3","timestamp":"2024-09-03T10:38:46.252705","nav_name":"api_owasp_sen_data","action":"组件启动","params":"数据窗口函数-3, 定时任务函数-0","operate_result":"信息","failed_reason":""}
{"user":"x_finder3","timestamp":"2024-09-03T10:38:46.252886","nav_name":"api_owasp_sen_data","action":"事件主循环","params":"PubShm 建立中","operate_result":"信息","failed_reason":""}
{"user":"x_finder3","timestamp":"2024-09-03T10:38:46.253568","nav_name":"api_owasp_sen_data","action":"启动出错","params":"PubShm","operate_result":"错误","failed_reason":"[Errno 2] No such file or directory: '\/xlink_shm_httpub_2.pub'"}
{"user":"x_finder3","timestamp":"2024-09-03T10:38:46.254810","nav_name":"api_owasp_sen_data","action":"主函数","params":"异常退出PubShm:[Errno 2] No such file or directory: '\/xlink_shm_httpub_2.pub'","operate_result":"错误","failed_reason":"错误跟踪Traceback (most recent call last):\n  File \"xlink_core.pyx\", line 820, in xlink_core.Main5\n  File \"\/opt\/fbi-base\/lib\/python3.11\/multiprocessing\/shared_memory.py\", line 104, in __init__\n    self._fd = _posixshmem.shm_open(\n               ^^^^^^^^^^^^^^^^^^^^^\nFileNotFoundError: [Errno 2] No such file or directory: '\/xlink_shm_httpub_2.pub'\n\nDuring handling of the above exception, another exception occurred:\n\nTraceback (most recent call last):\n  File \"xlink_core.pyx\", line 1453, in xlink_core.x_start\n  File \"xlink_core.pyx\", line 830, in xlink_core.Main5\nException: PubShm:[Errno 2] No such file or directory: '\/xlink_shm_httpub_2.pub'\n"}
{"user":"x_finder3","timestamp":"2024-09-03T10:38:46.254955","nav_name":"api_owasp_sen_data","action":"停止","params":"二阶段开始停止!","operate_result":"信息","failed_reason":""}
{"user":"x_finder3","timestamp":"2024-09-03T10:38:46.255061","nav_name":"api_owasp_sen_data","action":"停止","params":"三阶段开始停止!","operate_result":"信息","failed_reason":""}
{"user":"x_finder3","timestamp":"2024-09-03T10:38:46.352572","nav_name":"api_owasp_sen_data","action":"停止","params":"数据窗口任务停止完毕!","operate_result":"信息","failed_reason":""}
{"user":"x_finder3","timestamp":"2024-09-03T10:38:46.752734","nav_name":"api_owasp_sen_data","action":"停止","params":"系统定时任务st_10s停止完毕!","operate_result":"信息","failed_reason":""}
{"user":"x_finder3","timestamp":"2024-09-03T10:38:51.252835","nav_name":"api_owasp_sen_data","action":"监控程序","params":"printf::收到消息退出","operate_result":"信息","failed_reason":""}
{"user":"x_finder3","timestamp":"2024-09-03T10:38:51.253382","nav_name":"api_owasp_sen_data","action":"停止","params":"全部停止完毕!","operate_result":"信息","failed_reason":""}
{"user":"x_finder3","timestamp":"2024-09-03T10:39:46.501421","nav_name":"api_owasp_sen_data","action":"启动参数","params":"shm_name: httpub,count: 8,","operate_result":"信息","failed_reason":""}
{"user":"x_finder3","timestamp":"2024-09-03T10:39:46.508650","nav_name":"api_owasp_sen_data","action":"组件启动","params":"开始验证授权....","operate_result":"信息","failed_reason":""}
{"user":"x_finder3","timestamp":"2024-09-03T10:39:46.509097","nav_name":"api_owasp_sen_data","action":"组件启动","params":"授权验证完成!","operate_result":"信息","failed_reason":""}
{"user":"x_finder3","timestamp":"2024-09-03T10:39:46.510412","nav_name":"api_owasp_sen_data","action":"组件启动","params":"数据窗口函数-3, 定时任务函数-0","operate_result":"信息","failed_reason":""}
{"user":"x_finder3","timestamp":"2024-09-03T10:39:46.510587","nav_name":"api_owasp_sen_data","action":"事件主循环","params":"PubShm 建立中","operate_result":"信息","failed_reason":""}
{"user":"x_finder3","timestamp":"2024-09-03T10:39:46.511357","nav_name":"api_owasp_sen_data","action":"启动出错","params":"PubShm","operate_result":"错误","failed_reason":"[Errno 2] No such file or directory: '\/xlink_shm_httpub_2.pub'"}
{"user":"x_finder3","timestamp":"2024-09-03T10:39:46.512724","nav_name":"api_owasp_sen_data","action":"主函数","params":"异常退出PubShm:[Errno 2] No such file or directory: '\/xlink_shm_httpub_2.pub'","operate_result":"错误","failed_reason":"错误跟踪Traceback (most recent call last):\n  File \"xlink_core.pyx\", line 820, in xlink_core.Main5\n  File \"\/opt\/fbi-base\/lib\/python3.11\/multiprocessing\/shared_memory.py\", line 104, in __init__\n    self._fd = _posixshmem.shm_open(\n               ^^^^^^^^^^^^^^^^^^^^^\nFileNotFoundError: [Errno 2] No such file or directory: '\/xlink_shm_httpub_2.pub'\n\nDuring handling of the above exception, another exception occurred:\n\nTraceback (most recent call last):\n  File \"xlink_core.pyx\", line 1453, in xlink_core.x_start\n  File \"xlink_core.pyx\", line 830, in xlink_core.Main5\nException: PubShm:[Errno 2] No such file or directory: '\/xlink_shm_httpub_2.pub'\n"}
{"user":"x_finder3","timestamp":"2024-09-03T10:39:46.512876","nav_name":"api_owasp_sen_data","action":"停止","params":"二阶段开始停止!","operate_result":"信息","failed_reason":""}
{"user":"x_finder3","timestamp":"2024-09-03T10:39:46.512983","nav_name":"api_owasp_sen_data","action":"停止","params":"三阶段开始停止!","operate_result":"信息","failed_reason":""}
{"user":"x_finder3","timestamp":"2024-09-03T10:39:46.610244","nav_name":"api_owasp_sen_data","action":"停止","params":"数据窗口任务停止完毕!","operate_result":"信息","failed_reason":""}
{"user":"x_finder3","timestamp":"2024-09-03T10:39:47.010400","nav_name":"api_owasp_sen_data","action":"停止","params":"系统定时任务st_10s停止完毕!","operate_result":"信息","failed_reason":""}
{"user":"x_finder3","timestamp":"2024-09-03T10:39:51.510525","nav_name":"api_owasp_sen_data","action":"监控程序","params":"printf::收到消息退出","operate_result":"信息","failed_reason":""}
{"user":"x_finder3","timestamp":"2024-09-03T10:39:51.511101","nav_name":"api_owasp_sen_data","action":"停止","params":"全部停止完毕!","operate_result":"信息","failed_reason":""}
{"user":"x_finder3","timestamp":"2024-09-03T10:40:46.712271","nav_name":"api_owasp_sen_data","action":"启动参数","params":"shm_name: httpub,count: 8,","operate_result":"信息","failed_reason":""}
{"user":"x_finder3","timestamp":"2024-09-03T10:40:46.720120","nav_name":"api_owasp_sen_data","action":"组件启动","params":"开始验证授权....","operate_result":"信息","failed_reason":""}
{"user":"x_finder3","timestamp":"2024-09-03T10:40:46.720541","nav_name":"api_owasp_sen_data","action":"组件启动","params":"授权验证完成!","operate_result":"信息","failed_reason":""}
{"user":"x_finder3","timestamp":"2024-09-03T10:40:46.721773","nav_name":"api_owasp_sen_data","action":"组件启动","params":"数据窗口函数-3, 定时任务函数-0","operate_result":"信息","failed_reason":""}
{"user":"x_finder3","timestamp":"2024-09-03T10:40:46.721919","nav_name":"api_owasp_sen_data","action":"事件主循环","params":"PubShm 建立中","operate_result":"信息","failed_reason":""}
{"user":"x_finder3","timestamp":"2024-09-03T10:40:46.722546","nav_name":"api_owasp_sen_data","action":"启动出错","params":"PubShm","operate_result":"错误","failed_reason":"[Errno 2] No such file or directory: '\/xlink_shm_httpub_2.pub'"}
{"user":"x_finder3","timestamp":"2024-09-03T10:40:46.723840","nav_name":"api_owasp_sen_data","action":"主函数","params":"异常退出PubShm:[Errno 2] No such file or directory: '\/xlink_shm_httpub_2.pub'","operate_result":"错误","failed_reason":"错误跟踪Traceback (most recent call last):\n  File \"xlink_core.pyx\", line 820, in xlink_core.Main5\n  File \"\/opt\/fbi-base\/lib\/python3.11\/multiprocessing\/shared_memory.py\", line 104, in __init__\n    self._fd = _posixshmem.shm_open(\n               ^^^^^^^^^^^^^^^^^^^^^\nFileNotFoundError: [Errno 2] No such file or directory: '\/xlink_shm_httpub_2.pub'\n\nDuring handling of the above exception, another exception occurred:\n\nTraceback (most recent call last):\n  File \"xlink_core.pyx\", line 1453, in xlink_core.x_start\n  File \"xlink_core.pyx\", line 830, in xlink_core.Main5\nException: PubShm:[Errno 2] No such file or directory: '\/xlink_shm_httpub_2.pub'\n"}
{"user":"x_finder3","timestamp":"2024-09-03T10:40:46.723964","nav_name":"api_owasp_sen_data","action":"停止","params":"二阶段开始停止!","operate_result":"信息","failed_reason":""}
{"user":"x_finder3","timestamp":"2024-09-03T10:40:46.724061","nav_name":"api_owasp_sen_data","action":"停止","params":"三阶段开始停止!","operate_result":"信息","failed_reason":""}
{"user":"x_finder3","timestamp":"2024-09-03T10:40:46.821583","nav_name":"api_owasp_sen_data","action":"停止","params":"数据窗口任务停止完毕!","operate_result":"信息","failed_reason":""}
{"user":"x_finder3","timestamp":"2024-09-03T10:40:47.221725","nav_name":"api_owasp_sen_data","action":"停止","params":"系统定时任务st_10s停止完毕!","operate_result":"信息","failed_reason":""}
{"user":"x_finder3","timestamp":"2024-09-03T10:40:51.721896","nav_name":"api_owasp_sen_data","action":"监控程序","params":"printf::收到消息退出","operate_result":"信息","failed_reason":""}
{"user":"x_finder3","timestamp":"2024-09-03T10:40:51.722396","nav_name":"api_owasp_sen_data","action":"停止","params":"全部停止完毕!","operate_result":"信息","failed_reason":""}
{"user":"x_finder3","timestamp":"2024-09-03T10:41:46.924710","nav_name":"api_owasp_sen_data","action":"启动参数","params":"shm_name: httpub,count: 8,","operate_result":"信息","failed_reason":""}
{"user":"x_finder3","timestamp":"2024-09-03T10:41:46.932063","nav_name":"api_owasp_sen_data","action":"组件启动","params":"开始验证授权....","operate_result":"信息","failed_reason":""}
{"user":"x_finder3","timestamp":"2024-09-03T10:41:46.932443","nav_name":"api_owasp_sen_data","action":"组件启动","params":"授权验证完成!","operate_result":"信息","failed_reason":""}
{"user":"x_finder3","timestamp":"2024-09-03T10:41:46.933519","nav_name":"api_owasp_sen_data","action":"组件启动","params":"数据窗口函数-3, 定时任务函数-0","operate_result":"信息","failed_reason":""}
{"user":"x_finder3","timestamp":"2024-09-03T10:41:46.933692","nav_name":"api_owasp_sen_data","action":"事件主循环","params":"PubShm 建立中","operate_result":"信息","failed_reason":""}
{"user":"x_finder3","timestamp":"2024-09-03T10:41:46.934323","nav_name":"api_owasp_sen_data","action":"启动出错","params":"PubShm","operate_result":"错误","failed_reason":"[Errno 2] No such file or directory: '\/xlink_shm_httpub_2.pub'"}
{"user":"x_finder3","timestamp":"2024-09-03T10:41:46.935393","nav_name":"api_owasp_sen_data","action":"主函数","params":"异常退出PubShm:[Errno 2] No such file or directory: '\/xlink_shm_httpub_2.pub'","operate_result":"错误","failed_reason":"错误跟踪Traceback (most recent call last):\n  File \"xlink_core.pyx\", line 820, in xlink_core.Main5\n  File \"\/opt\/fbi-base\/lib\/python3.11\/multiprocessing\/shared_memory.py\", line 104, in __init__\n    self._fd = _posixshmem.shm_open(\n               ^^^^^^^^^^^^^^^^^^^^^\nFileNotFoundError: [Errno 2] No such file or directory: '\/xlink_shm_httpub_2.pub'\n\nDuring handling of the above exception, another exception occurred:\n\nTraceback (most recent call last):\n  File \"xlink_core.pyx\", line 1453, in xlink_core.x_start\n  File \"xlink_core.pyx\", line 830, in xlink_core.Main5\nException: PubShm:[Errno 2] No such file or directory: '\/xlink_shm_httpub_2.pub'\n"}
{"user":"x_finder3","timestamp":"2024-09-03T10:41:46.935511","nav_name":"api_owasp_sen_data","action":"停止","params":"二阶段开始停止!","operate_result":"信息","failed_reason":""}
{"user":"x_finder3","timestamp":"2024-09-03T10:41:46.935611","nav_name":"api_owasp_sen_data","action":"停止","params":"三阶段开始停止!","operate_result":"信息","failed_reason":""}
{"user":"x_finder3","timestamp":"2024-09-03T10:41:47.033369","nav_name":"api_owasp_sen_data","action":"停止","params":"数据窗口任务停止完毕!","operate_result":"信息","failed_reason":""}
{"user":"x_finder3","timestamp":"2024-09-03T10:41:47.433489","nav_name":"api_owasp_sen_data","action":"停止","params":"系统定时任务st_10s停止完毕!","operate_result":"信息","failed_reason":""}
{"user":"x_finder3","timestamp":"2024-09-03T10:41:51.933649","nav_name":"api_owasp_sen_data","action":"监控程序","params":"printf::收到消息退出","operate_result":"信息","failed_reason":""}
{"user":"x_finder3","timestamp":"2024-09-03T10:41:51.934157","nav_name":"api_owasp_sen_data","action":"停止","params":"全部停止完毕!","operate_result":"信息","failed_reason":""}
{"user":"x_finder3","timestamp":"2024-09-03T10:42:46.822863","nav_name":"api_owasp_sen_data","action":"启动参数","params":"shm_name: httpub,count: 8,","operate_result":"信息","failed_reason":""}
{"user":"x_finder3","timestamp":"2024-09-03T10:42:46.828369","nav_name":"api_owasp_sen_data","action":"组件启动","params":"开始验证授权....","operate_result":"信息","failed_reason":""}
{"user":"x_finder3","timestamp":"2024-09-03T10:42:46.828707","nav_name":"api_owasp_sen_data","action":"组件启动","params":"授权验证完成!","operate_result":"信息","failed_reason":""}
{"user":"x_finder3","timestamp":"2024-09-03T10:42:46.829569","nav_name":"api_owasp_sen_data","action":"组件启动","params":"数据窗口函数-3, 定时任务函数-0","operate_result":"信息","failed_reason":""}
{"user":"x_finder3","timestamp":"2024-09-03T10:42:46.829715","nav_name":"api_owasp_sen_data","action":"事件主循环","params":"PubShm 建立中","operate_result":"信息","failed_reason":""}
{"user":"x_finder3","timestamp":"2024-09-03T10:42:46.830282","nav_name":"api_owasp_sen_data","action":"启动出错","params":"PubShm","operate_result":"错误","failed_reason":"[Errno 2] No such file or directory: '\/xlink_shm_httpub_2.pub'"}
{"user":"x_finder3","timestamp":"2024-09-03T10:42:46.831019","nav_name":"api_owasp_sen_data","action":"主函数","params":"异常退出PubShm:[Errno 2] No such file or directory: '\/xlink_shm_httpub_2.pub'","operate_result":"错误","failed_reason":"错误跟踪Traceback (most recent call last):\n  File \"xlink_core.pyx\", line 820, in xlink_core.Main5\n  File \"\/opt\/fbi-base\/lib\/python3.11\/multiprocessing\/shared_memory.py\", line 104, in __init__\n    self._fd = _posixshmem.shm_open(\n               ^^^^^^^^^^^^^^^^^^^^^\nFileNotFoundError: [Errno 2] No such file or directory: '\/xlink_shm_httpub_2.pub'\n\nDuring handling of the above exception, another exception occurred:\n\nTraceback (most recent call last):\n  File \"xlink_core.pyx\", line 1453, in xlink_core.x_start\n  File \"xlink_core.pyx\", line 830, in xlink_core.Main5\nException: PubShm:[Errno 2] No such file or directory: '\/xlink_shm_httpub_2.pub'\n"}
{"user":"x_finder3","timestamp":"2024-09-03T10:42:46.831136","nav_name":"api_owasp_sen_data","action":"停止","params":"二阶段开始停止!","operate_result":"信息","failed_reason":""}
{"user":"x_finder3","timestamp":"2024-09-03T10:42:46.831234","nav_name":"api_owasp_sen_data","action":"停止","params":"三阶段开始停止!","operate_result":"信息","failed_reason":""}
{"user":"x_finder3","timestamp":"2024-09-03T10:42:46.929456","nav_name":"api_owasp_sen_data","action":"停止","params":"数据窗口任务停止完毕!","operate_result":"信息","failed_reason":""}
{"user":"x_finder3","timestamp":"2024-09-03T10:42:47.329578","nav_name":"api_owasp_sen_data","action":"停止","params":"系统定时任务st_10s停止完毕!","operate_result":"信息","failed_reason":""}
{"user":"x_finder3","timestamp":"2024-09-03T10:42:51.829698","nav_name":"api_owasp_sen_data","action":"监控程序","params":"printf::收到消息退出","operate_result":"信息","failed_reason":""}
{"user":"x_finder3","timestamp":"2024-09-03T10:42:51.830121","nav_name":"api_owasp_sen_data","action":"停止","params":"全部停止完毕!","operate_result":"信息","failed_reason":""}
{"user":"x_finder3","timestamp":"2024-09-03T10:43:47.260848","nav_name":"api_owasp_sen_data","action":"启动参数","params":"shm_name: httpub,count: 8,","operate_result":"信息","failed_reason":""}
{"user":"x_finder3","timestamp":"2024-09-03T10:43:47.266517","nav_name":"api_owasp_sen_data","action":"组件启动","params":"开始验证授权....","operate_result":"信息","failed_reason":""}
{"user":"x_finder3","timestamp":"2024-09-03T10:43:47.266892","nav_name":"api_owasp_sen_data","action":"组件启动","params":"授权验证完成!","operate_result":"信息","failed_reason":""}
{"user":"x_finder3","timestamp":"2024-09-03T10:43:47.267967","nav_name":"api_owasp_sen_data","action":"组件启动","params":"数据窗口函数-3, 定时任务函数-0","operate_result":"信息","failed_reason":""}
{"user":"x_finder3","timestamp":"2024-09-03T10:43:47.268171","nav_name":"api_owasp_sen_data","action":"事件主循环","params":"PubShm 建立中","operate_result":"信息","failed_reason":""}
{"user":"x_finder3","timestamp":"2024-09-03T10:43:47.268785","nav_name":"api_owasp_sen_data","action":"启动出错","params":"PubShm","operate_result":"错误","failed_reason":"[Errno 2] No such file or directory: '\/xlink_shm_httpub_2.pub'"}
{"user":"x_finder3","timestamp":"2024-09-03T10:43:47.269558","nav_name":"api_owasp_sen_data","action":"主函数","params":"异常退出PubShm:[Errno 2] No such file or directory: '\/xlink_shm_httpub_2.pub'","operate_result":"错误","failed_reason":"错误跟踪Traceback (most recent call last):\n  File \"xlink_core.pyx\", line 820, in xlink_core.Main5\n  File \"\/opt\/fbi-base\/lib\/python3.11\/multiprocessing\/shared_memory.py\", line 104, in __init__\n    self._fd = _posixshmem.shm_open(\n               ^^^^^^^^^^^^^^^^^^^^^\nFileNotFoundError: [Errno 2] No such file or directory: '\/xlink_shm_httpub_2.pub'\n\nDuring handling of the above exception, another exception occurred:\n\nTraceback (most recent call last):\n  File \"xlink_core.pyx\", line 1453, in xlink_core.x_start\n  File \"xlink_core.pyx\", line 830, in xlink_core.Main5\nException: PubShm:[Errno 2] No such file or directory: '\/xlink_shm_httpub_2.pub'\n"}
{"user":"x_finder3","timestamp":"2024-09-03T10:43:47.269695","nav_name":"api_owasp_sen_data","action":"停止","params":"二阶段开始停止!","operate_result":"信息","failed_reason":""}
{"user":"x_finder3","timestamp":"2024-09-03T10:43:47.269802","nav_name":"api_owasp_sen_data","action":"停止","params":"三阶段开始停止!","operate_result":"信息","failed_reason":""}
{"user":"x_finder3","timestamp":"2024-09-03T10:43:47.367719","nav_name":"api_owasp_sen_data","action":"停止","params":"数据窗口任务停止完毕!","operate_result":"信息","failed_reason":""}
{"user":"x_finder3","timestamp":"2024-09-03T10:43:47.767929","nav_name":"api_owasp_sen_data","action":"停止","params":"系统定时任务st_10s停止完毕!","operate_result":"信息","failed_reason":""}
{"user":"x_finder3","timestamp":"2024-09-03T10:43:52.268078","nav_name":"api_owasp_sen_data","action":"监控程序","params":"printf::收到消息退出","operate_result":"信息","failed_reason":""}
{"user":"x_finder3","timestamp":"2024-09-03T10:43:52.268482","nav_name":"api_owasp_sen_data","action":"停止","params":"全部停止完毕!","operate_result":"信息","failed_reason":""}
{"user":"x_finder3","timestamp":"2024-09-03T10:44:47.285252","nav_name":"api_owasp_sen_data","action":"启动参数","params":"shm_name: httpub,count: 8,","operate_result":"信息","failed_reason":""}
{"user":"x_finder3","timestamp":"2024-09-03T10:44:47.292640","nav_name":"api_owasp_sen_data","action":"组件启动","params":"开始验证授权....","operate_result":"信息","failed_reason":""}
{"user":"x_finder3","timestamp":"2024-09-03T10:44:47.293034","nav_name":"api_owasp_sen_data","action":"组件启动","params":"授权验证完成!","operate_result":"信息","failed_reason":""}
{"user":"x_finder3","timestamp":"2024-09-03T10:44:47.294182","nav_name":"api_owasp_sen_data","action":"组件启动","params":"数据窗口函数-3, 定时任务函数-0","operate_result":"信息","failed_reason":""}
{"user":"x_finder3","timestamp":"2024-09-03T10:44:47.294354","nav_name":"api_owasp_sen_data","action":"事件主循环","params":"PubShm 建立中","operate_result":"信息","failed_reason":""}
{"user":"x_finder3","timestamp":"2024-09-03T10:44:47.295019","nav_name":"api_owasp_sen_data","action":"启动出错","params":"PubShm","operate_result":"错误","failed_reason":"[Errno 2] No such file or directory: '\/xlink_shm_httpub_2.pub'"}
{"user":"x_finder3","timestamp":"2024-09-03T10:44:47.296242","nav_name":"api_owasp_sen_data","action":"主函数","params":"异常退出PubShm:[Errno 2] No such file or directory: '\/xlink_shm_httpub_2.pub'","operate_result":"错误","failed_reason":"错误跟踪Traceback (most recent call last):\n  File \"xlink_core.pyx\", line 820, in xlink_core.Main5\n  File \"\/opt\/fbi-base\/lib\/python3.11\/multiprocessing\/shared_memory.py\", line 104, in __init__\n    self._fd = _posixshmem.shm_open(\n               ^^^^^^^^^^^^^^^^^^^^^\nFileNotFoundError: [Errno 2] No such file or directory: '\/xlink_shm_httpub_2.pub'\n\nDuring handling of the above exception, another exception occurred:\n\nTraceback (most recent call last):\n  File \"xlink_core.pyx\", line 1453, in xlink_core.x_start\n  File \"xlink_core.pyx\", line 830, in xlink_core.Main5\nException: PubShm:[Errno 2] No such file or directory: '\/xlink_shm_httpub_2.pub'\n"}
{"user":"x_finder3","timestamp":"2024-09-03T10:44:47.296405","nav_name":"api_owasp_sen_data","action":"停止","params":"二阶段开始停止!","operate_result":"信息","failed_reason":""}
{"user":"x_finder3","timestamp":"2024-09-03T10:44:47.296507","nav_name":"api_owasp_sen_data","action":"停止","params":"三阶段开始停止!","operate_result":"信息","failed_reason":""}
{"user":"x_finder3","timestamp":"2024-09-03T10:44:47.394080","nav_name":"api_owasp_sen_data","action":"停止","params":"数据窗口任务停止完毕!","operate_result":"信息","failed_reason":""}
{"user":"x_finder3","timestamp":"2024-09-03T10:44:47.794176","nav_name":"api_owasp_sen_data","action":"停止","params":"系统定时任务st_10s停止完毕!","operate_result":"信息","failed_reason":""}
{"user":"x_finder3","timestamp":"2024-09-03T10:44:52.294291","nav_name":"api_owasp_sen_data","action":"监控程序","params":"printf::收到消息退出","operate_result":"信息","failed_reason":""}
{"user":"x_finder3","timestamp":"2024-09-03T10:44:52.294770","nav_name":"api_owasp_sen_data","action":"停止","params":"全部停止完毕!","operate_result":"信息","failed_reason":""}
{"user":"x_finder3","timestamp":"2024-09-03T10:45:47.846127","nav_name":"api_owasp_sen_data","action":"启动参数","params":"shm_name: httpub,count: 8,","operate_result":"信息","failed_reason":""}
{"user":"x_finder3","timestamp":"2024-09-03T10:45:47.854092","nav_name":"api_owasp_sen_data","action":"组件启动","params":"开始验证授权....","operate_result":"信息","failed_reason":""}
{"user":"x_finder3","timestamp":"2024-09-03T10:45:47.854553","nav_name":"api_owasp_sen_data","action":"组件启动","params":"授权验证完成!","operate_result":"信息","failed_reason":""}
{"user":"x_finder3","timestamp":"2024-09-03T10:45:47.855768","nav_name":"api_owasp_sen_data","action":"组件启动","params":"数据窗口函数-3, 定时任务函数-0","operate_result":"信息","failed_reason":""}
{"user":"x_finder3","timestamp":"2024-09-03T10:45:47.855957","nav_name":"api_owasp_sen_data","action":"事件主循环","params":"PubShm 建立中","operate_result":"信息","failed_reason":""}
{"user":"x_finder3","timestamp":"2024-09-03T10:45:47.856882","nav_name":"api_owasp_sen_data","action":"启动出错","params":"PubShm","operate_result":"错误","failed_reason":"[Errno 2] No such file or directory: '\/xlink_shm_httpub_2.pub'"}
{"user":"x_finder3","timestamp":"2024-09-03T10:45:47.858454","nav_name":"api_owasp_sen_data","action":"主函数","params":"异常退出PubShm:[Errno 2] No such file or directory: '\/xlink_shm_httpub_2.pub'","operate_result":"错误","failed_reason":"错误跟踪Traceback (most recent call last):\n  File \"xlink_core.pyx\", line 820, in xlink_core.Main5\n  File \"\/opt\/fbi-base\/lib\/python3.11\/multiprocessing\/shared_memory.py\", line 104, in __init__\n    self._fd = _posixshmem.shm_open(\n               ^^^^^^^^^^^^^^^^^^^^^\nFileNotFoundError: [Errno 2] No such file or directory: '\/xlink_shm_httpub_2.pub'\n\nDuring handling of the above exception, another exception occurred:\n\nTraceback (most recent call last):\n  File \"xlink_core.pyx\", line 1453, in xlink_core.x_start\n  File \"xlink_core.pyx\", line 830, in xlink_core.Main5\nException: PubShm:[Errno 2] No such file or directory: '\/xlink_shm_httpub_2.pub'\n"}
{"user":"x_finder3","timestamp":"2024-09-03T10:45:47.858629","nav_name":"api_owasp_sen_data","action":"停止","params":"二阶段开始停止!","operate_result":"信息","failed_reason":""}
{"user":"x_finder3","timestamp":"2024-09-03T10:45:47.858736","nav_name":"api_owasp_sen_data","action":"停止","params":"三阶段开始停止!","operate_result":"信息","failed_reason":""}
{"user":"x_finder3","timestamp":"2024-09-03T10:45:47.955578","nav_name":"api_owasp_sen_data","action":"停止","params":"数据窗口任务停止完毕!","operate_result":"信息","failed_reason":""}
{"user":"x_finder3","timestamp":"2024-09-03T10:45:48.355711","nav_name":"api_owasp_sen_data","action":"停止","params":"系统定时任务st_10s停止完毕!","operate_result":"信息","failed_reason":""}
{"user":"x_finder3","timestamp":"2024-09-03T10:45:52.855888","nav_name":"api_owasp_sen_data","action":"监控程序","params":"printf::收到消息退出","operate_result":"信息","failed_reason":""}
{"user":"x_finder3","timestamp":"2024-09-03T10:45:52.856466","nav_name":"api_owasp_sen_data","action":"停止","params":"全部停止完毕!","operate_result":"信息","failed_reason":""}
{"user":"x_finder3","timestamp":"2024-09-03T10:46:47.841552","nav_name":"api_owasp_sen_data","action":"启动参数","params":"shm_name: httpub,count: 8,","operate_result":"信息","failed_reason":""}
{"user":"x_finder3","timestamp":"2024-09-03T10:46:47.847299","nav_name":"api_owasp_sen_data","action":"组件启动","params":"开始验证授权....","operate_result":"信息","failed_reason":""}
{"user":"x_finder3","timestamp":"2024-09-03T10:46:47.847660","nav_name":"api_owasp_sen_data","action":"组件启动","params":"授权验证完成!","operate_result":"信息","failed_reason":""}
{"user":"x_finder3","timestamp":"2024-09-03T10:46:47.848517","nav_name":"api_owasp_sen_data","action":"组件启动","params":"数据窗口函数-3, 定时任务函数-0","operate_result":"信息","failed_reason":""}
{"user":"x_finder3","timestamp":"2024-09-03T10:46:47.848662","nav_name":"api_owasp_sen_data","action":"事件主循环","params":"PubShm 建立中","operate_result":"信息","failed_reason":""}
{"user":"x_finder3","timestamp":"2024-09-03T10:46:47.849223","nav_name":"api_owasp_sen_data","action":"启动出错","params":"PubShm","operate_result":"错误","failed_reason":"[Errno 2] No such file or directory: '\/xlink_shm_httpub_2.pub'"}
{"user":"x_finder3","timestamp":"2024-09-03T10:46:47.849953","nav_name":"api_owasp_sen_data","action":"主函数","params":"异常退出PubShm:[Errno 2] No such file or directory: '\/xlink_shm_httpub_2.pub'","operate_result":"错误","failed_reason":"错误跟踪Traceback (most recent call last):\n  File \"xlink_core.pyx\", line 820, in xlink_core.Main5\n  File \"\/opt\/fbi-base\/lib\/python3.11\/multiprocessing\/shared_memory.py\", line 104, in __init__\n    self._fd = _posixshmem.shm_open(\n               ^^^^^^^^^^^^^^^^^^^^^\nFileNotFoundError: [Errno 2] No such file or directory: '\/xlink_shm_httpub_2.pub'\n\nDuring handling of the above exception, another exception occurred:\n\nTraceback (most recent call last):\n  File \"xlink_core.pyx\", line 1453, in xlink_core.x_start\n  File \"xlink_core.pyx\", line 830, in xlink_core.Main5\nException: PubShm:[Errno 2] No such file or directory: '\/xlink_shm_httpub_2.pub'\n"}
{"user":"x_finder3","timestamp":"2024-09-03T10:46:47.850071","nav_name":"api_owasp_sen_data","action":"停止","params":"二阶段开始停止!","operate_result":"信息","failed_reason":""}
{"user":"x_finder3","timestamp":"2024-09-03T10:46:47.850171","nav_name":"api_owasp_sen_data","action":"停止","params":"三阶段开始停止!","operate_result":"信息","failed_reason":""}
{"user":"x_finder3","timestamp":"2024-09-03T10:46:47.948403","nav_name":"api_owasp_sen_data","action":"停止","params":"数据窗口任务停止完毕!","operate_result":"信息","failed_reason":""}
{"user":"x_finder3","timestamp":"2024-09-03T10:46:48.348526","nav_name":"api_owasp_sen_data","action":"停止","params":"系统定时任务st_10s停止完毕!","operate_result":"信息","failed_reason":""}
{"user":"x_finder3","timestamp":"2024-09-03T10:46:52.848633","nav_name":"api_owasp_sen_data","action":"监控程序","params":"printf::收到消息退出","operate_result":"信息","failed_reason":""}
{"user":"x_finder3","timestamp":"2024-09-03T10:46:52.849049","nav_name":"api_owasp_sen_data","action":"停止","params":"全部停止完毕!","operate_result":"信息","failed_reason":""}
{"user":"x_finder3","timestamp":"2024-09-03T10:47:48.281323","nav_name":"api_owasp_sen_data","action":"启动参数","params":"shm_name: httpub,count: 8,","operate_result":"信息","failed_reason":""}
{"user":"x_finder3","timestamp":"2024-09-03T10:47:48.286869","nav_name":"api_owasp_sen_data","action":"组件启动","params":"开始验证授权....","operate_result":"信息","failed_reason":""}
{"user":"x_finder3","timestamp":"2024-09-03T10:47:48.287210","nav_name":"api_owasp_sen_data","action":"组件启动","params":"授权验证完成!","operate_result":"信息","failed_reason":""}
{"user":"x_finder3","timestamp":"2024-09-03T10:47:48.288047","nav_name":"api_owasp_sen_data","action":"组件启动","params":"数据窗口函数-3, 定时任务函数-0","operate_result":"信息","failed_reason":""}
{"user":"x_finder3","timestamp":"2024-09-03T10:47:48.288191","nav_name":"api_owasp_sen_data","action":"事件主循环","params":"PubShm 建立中","operate_result":"信息","failed_reason":""}
{"user":"x_finder3","timestamp":"2024-09-03T10:47:48.288748","nav_name":"api_owasp_sen_data","action":"启动出错","params":"PubShm","operate_result":"错误","failed_reason":"[Errno 2] No such file or directory: '\/xlink_shm_httpub_2.pub'"}
{"user":"x_finder3","timestamp":"2024-09-03T10:47:48.289450","nav_name":"api_owasp_sen_data","action":"主函数","params":"异常退出PubShm:[Errno 2] No such file or directory: '\/xlink_shm_httpub_2.pub'","operate_result":"错误","failed_reason":"错误跟踪Traceback (most recent call last):\n  File \"xlink_core.pyx\", line 820, in xlink_core.Main5\n  File \"\/opt\/fbi-base\/lib\/python3.11\/multiprocessing\/shared_memory.py\", line 104, in __init__\n    self._fd = _posixshmem.shm_open(\n               ^^^^^^^^^^^^^^^^^^^^^\nFileNotFoundError: [Errno 2] No such file or directory: '\/xlink_shm_httpub_2.pub'\n\nDuring handling of the above exception, another exception occurred:\n\nTraceback (most recent call last):\n  File \"xlink_core.pyx\", line 1453, in xlink_core.x_start\n  File \"xlink_core.pyx\", line 830, in xlink_core.Main5\nException: PubShm:[Errno 2] No such file or directory: '\/xlink_shm_httpub_2.pub'\n"}
{"user":"x_finder3","timestamp":"2024-09-03T10:47:48.289565","nav_name":"api_owasp_sen_data","action":"停止","params":"二阶段开始停止!","operate_result":"信息","failed_reason":""}
{"user":"x_finder3","timestamp":"2024-09-03T10:47:48.289664","nav_name":"api_owasp_sen_data","action":"停止","params":"三阶段开始停止!","operate_result":"信息","failed_reason":""}
{"user":"x_finder3","timestamp":"2024-09-03T10:47:48.387924","nav_name":"api_owasp_sen_data","action":"停止","params":"数据窗口任务停止完毕!","operate_result":"信息","failed_reason":""}
{"user":"x_finder3","timestamp":"2024-09-03T10:47:48.788042","nav_name":"api_owasp_sen_data","action":"停止","params":"系统定时任务st_10s停止完毕!","operate_result":"信息","failed_reason":""}
{"user":"x_finder3","timestamp":"2024-09-03T10:47:53.288169","nav_name":"api_owasp_sen_data","action":"监控程序","params":"printf::收到消息退出","operate_result":"信息","failed_reason":""}
{"user":"x_finder3","timestamp":"2024-09-03T10:47:53.288629","nav_name":"api_owasp_sen_data","action":"停止","params":"全部停止完毕!","operate_result":"信息","failed_reason":""}
